distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -139,9 +139,9 @@
 
   private static final File UNPACK_DIRECTORY = new File("target/exam");
 
-  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(10);
+  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(20);
```

Comment:
I thought 10 minutes was already way to much time. I think we need a different approach. What if we change REMOVE_ALL to `catalog:removeall -f` which will leave the geoname metacards alone. If some of the tests need to remove other non resource metacards then they could be updated to do something like `catalog:remove --cql "\\\"metacard-tags\\\" =  'mytype'"`. This way we only remove the small number of metacards created by the tests.